src/components/InputField/InputField.jsx

```diff
@@ -3,32 +3,48 @@ import "./input-field.scss";
 import { useState } from "react";
 import { AiFillEye, AiFillEyeInvisible } from "react-icons/ai";
 
-function InputField({ labelName, type, value, name, errorMessage, ...other }) {
+function InputField({
```

Comment:
Use arrow functions for your components

Reply:
Thanks for pointing out, will update it in next PR